llama.cpp

```diff
@@ -930,6 +930,8 @@ static const size_t kB = 1024;
 static const size_t MB = kB*kB;
 static const size_t GB = kB*kB*kB;
 
+const double EPSILON = 1e-9;
```

Comment:
This should be a local constant inside the operator definition

Reply:
Done